### DISCUSSION TITLE
Which ports should I enable in my local firewall

### DISCUSSION THREAD

**Discussion by Fluidetom** [FreeTubeApp/FreeTube#5084] [Q&A] [answered]
Hi,
I've been using the Freetube app on my Linux laptop for a while now.
I'd like my kids to experience YT without all the ads so I decided to self-host a Freetube instance in a Docket container on my Synology NAS.
Everything is working fine, and I can access FT via my NAS local IP, BUT none of the videos loads when the Synology FW is enabled.
As soon as I disable the local Synology FW, FT works like a charm, I can search and view videos just fine.
But when the FW is enabled, I can't search or view any video in FT which keeps sending me the message "API error".

Which outgoing ports should I open in my FW ?
I didn't find anything in the documentation and unfortunately, Synology doesn't logthe blocked FW attempts.

Thanks in advance

**Comment by absidue** [comment, marked as answer]
FreeTube is a desktop app, there are no official docker images. The FreeTube team recommends using the app as it was meant to be used, as an app that you install on the computer you want to use it on.
However if you do for some reason still want to continue along that questionable path, you will have to go and get support from whoever made that unofficial thing that you are using, as the FreeTube team can't provide support for things it didn't even make.

**Reply by Fluidetom** [reply]
Oops my bad, I thought FT was also available as an official container image.
Thanks for getting back to me anyway.
